app/actions/local/systems.ts

```diff
@@ -36,6 +38,7 @@ export async function storeConfigAndLicense(serverUrl: string, config: ClientCon
 
             if (systems.length) {
                 await operator.handleSystem({systems, prepareRecordsOnly: false});
+                DeviceEventEmitter.emit(Events.LICENSE_CHANGED, {serverUrl, license});
```

Comment:
Nice to have for the changes in this file:

The tests checking that the events are sent with the correct parameters.